contrib/hook.sh

```diff
@@ -8,6 +10,3 @@ CG_PATH=$(cat config.json | jq -r '.linux.cgroupsPath')
 CONTAINERID=$(echo "$CG_PATH" | cut -d: -f3)
 
 grep burst <<<$CG_PATH && echo max > /sys/fs/cgroup/kubepods.slice/*burst*/*/crio-$CONTAINERID*/memory.swap.max || :
```

Comment:
Why do we need the `|| :` ?

Reply:
Ah yes, not needed anymore (I used to use`-e` in this file)